exonum/src/helpers/fabric/mod.rs

```diff
@@ -207,6 +238,13 @@ impl Context {
                         continue;
                     }
                 }
+                ArgumentType::Named(_) if !arg.takes_value => {
+                    let occurrences = matches.occurrences_of(&arg.name);
+                    if occurrences > 0 {
+                        context.flags.insert(arg.name.to_owned(), occurrences);
```

Comment:
Below we have a method for this ? 
`pub fn set_flag_occurrences(&mut self, flag: &str, occurrences: u64)`